### PR TITLE
New design

### PR DESCRIPTION
**Pull request by secondsun**

### CODE REVIEW

File: getstarted/guides/index.markdown [hunk 1]
```diff
@@ -15,7 +15,8 @@ Designed to get you started with enterprise mobile development.
 > Learn how to make first steps in the mobile development on each of supported platforms
 
 <h4 id="coreAndroid">Android</h4>
-* [How to build AeroGear Android library](/docs/guides/aerogear-android/how-to-build-aerogear-android)
+* [Google's Android Tutorials](https://developer.android.com/training/index.html)
+* [AeroGear Android General information](http://localhost:4000/android/)
```
Comment (danielpassos) [inline getstarted/guides/index.markdown]: localhost? :broken_heart:

**Comment by secondsun** [comment]
@danielpassos Fixed

**Comment by danielpassos** [comment]
:+1:

**Comment by lfryc** [comment]
:+1: merging

**Comment by abstractj** [comment]
@secondsun I guess it was already landed here https://github.com/aerogear/aerogear.org/commit/4cf092498a97441c9622d9ee7ca8db7397ec526d. But not closed, feel free to reopen if I'm crazy.

**Pull request closed without merging**